bottles/frontend/ui/details-bottle.blp

```diff
@@ -230,6 +215,142 @@ template DetailsBottle : .AdwPreferencesPage {
 
   .AdwPreferencesGroup group_programs {
     title: _("Programs");
+
+    .AdwActionRow row_no_programs {
+      Box {
+        hexpand: true;
+
+        Label {
+          label: _("Use the Installers section or the \"Run executable\" button.");
+          wrap: false;
+
+          styles [
+            "dim-label",
+          ]
+        }
+      }
+    }
+    
+    .AdwPreferencesRow bottom_bar {
+      activatable: false;      
+      
+      Box {
+        height-request:45;
```

Comment:
don't do this

Reply:
![image](https://user-images.githubusercontent.com/22079654/198192903-815f4c85-7fff-4200-a019-a630fa430979.png)
it looks like this without `height-request` and `has-frame` for buttons not set.

Reply:
you replace it with .flat style and the css in discord

Reply:
a48e8cda046903ca3dfbbafd316bf153c09675f0